go.mod

```diff
@@ -5,6 +5,7 @@ require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/graphql-go/graphql v0.7.7
 	github.com/graphql-go/handler v0.2.3
+	github.com/joho/godotenv v1.3.0
```

Comment:
this lib is a super simple way to have env vars that can default to what is specified in a env file. It's fine for now but isn't a substitute for structured config.

Reply:
I've toyed with the idea of using a structured config like a TOML file but haven't explored it. If you have ideas here I'm all ears

Reply:
toml and yaml are both fine, people have religious beliefs about them and usually hate one or the other.

I think its overkill right now though; one big benefit of structured configs is the ability to get types (arrays and whatnot), and to create generic config structures for things (e.g. configuring plugins or some other form of dynamic modular code). It's probably better to be flexible right now and then pick something later when the problems are understood better.